<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMyWindow</class>
 <widget class="QWidget" name="CMyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CMyWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QWidget" name="titleBarWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>900</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>64</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#titleBarWidget
{
	background-color: rgb(237, 239, 245);
	/*background-color: rgb(255, 255, 255);*/
}

</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_7" native="true">
           <property name="minimumSize">
            <size>
             <width>167</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>167</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget#widget_7
{
	background-color: rgb(255, 255, 255);
	border-bottom:3px solid #505052;
}</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>25</number>
            </property>
            <property name="topMargin">
             <number>3</number>
            </property>
            <property name="rightMargin">
             <number>38</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="labelTitle">
              <property name="minimumSize">
               <size>
                <width>104</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>104</width>
                <height>27</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>方正姚体</family>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">image: url(:/style/blue/logotitle.png);</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget#widget_5
{
	background-color: rgba(255, 255, 255,0);
	border-bottom:3px solid;
	border-bottom-color: qlineargradient(spread:pad, x1:0, y1:0.001, x2:0, y2:1, stop:0 rgba(0, 0, 0, 20), stop:1 rgba(0, 0, 0, 0));

}</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>3</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_6" native="true">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QWidget#widget_6
{
	background-color: rgb(255, 255, 255);
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="spacing">
                <number>12</number>
               </property>
               <property name="leftMargin">
                <number>25</number>
               </property>
               <property name="topMargin">
                <number>3</number>
               </property>
               <property name="rightMargin">
                <number>25</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>330</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QWidget" name="widget_2" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>628</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>628</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <property name="spacing">
                   <number>25</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QWidget" name="widget_4" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>18</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>18</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <widget class="QPushButton" name="btnWjzs">
                     <property name="geometry">
                      <rect>
                       <x>0</x>
                       <y>20</y>
                       <width>16</width>
                       <height>16</height>
                      </rect>
                     </property>
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>16</width>
                       <height>16</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>16</width>
                       <height>16</height>
                      </size>
                     </property>
                     <property name="cursor">
                      <cursorShape>PointingHandCursor</cursorShape>
                     </property>
                     <property name="toolTip">
                      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;文件助手&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                     </property>
                     <property name="whatsThis">
                      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;下载列表&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">QPushButton
{
		background-image: url(:/style/blue/yunpan/header_icon_zhushou_n.png);
		background-position: center;
		background-repeat: no-repeat;
		border-radius: 3px;
}

QPushButton::hover
{
		background-image: url(:/style/blue/yunpan/header_icon_zhushou_h.png);
		background-position: center;
		background-repeat: no-repeat;
		border-radius: 3px;
}</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                    <widget class="QLabel" name="wjzsMsgLabel">
                     <property name="geometry">
                      <rect>
                       <x>10</x>
                       <y>16</y>
                       <width>8</width>
                       <height>8</height>
                      </rect>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>8</width>
                       <height>8</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>8</width>
                       <height>8</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">border:1px solid #fc4c45;
border-radius: 4px;
background-color: #fc4c45;</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </widget>
                  </item>
                  <item>
                   <widget class="QWidget" name="widget_3" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>18</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>18</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <widget class="QPushButton" name="btnTzzx">
                     <property name="geometry">
                      <rect>
                       <x>0</x>
                       <y>20</y>
                       <width>16</width>
                       <height>16</height>
                      </rect>
                     </property>
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>16</width>
                       <height>16</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>16</width>
                       <height>16</height>
                      </size>
                     </property>
                     <property name="cursor">
                      <cursorShape>PointingHandCursor</cursorShape>
                     </property>
                     <property name="toolTip">
                      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;通知中心&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">
/*btnMessage*/
QPushButton#btnTzzx
{
    background-image: url(:/style/blue/yunpan/header_icon_tixing_n.png);
    background-position: center;
    background-repeat: no-repeat;
    border-radius: 3px;
}

QPushButton#btnTzzx:hover
{
    background-image: url(:/style/blue/yunpan/header_icon_tixing_h.png);
    background-position: center;
    background-repeat: no-repeat;
    border-radius: 3px;
}</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                    <widget class="QLabel" name="tzzxMsgLabel">
                     <property name="geometry">
                      <rect>
                       <x>10</x>
                       <y>16</y>
                       <width>8</width>
                       <height>8</height>
                      </rect>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>8</width>
                       <height>8</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>8</width>
                       <height>8</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">border:1px solid #fc4c45;
border-radius: 4px;
background-color: #fc4c45;</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btnSetting">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>16</width>
                      <height>16</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16</width>
                      <height>16</height>
                     </size>
                    </property>
                    <property name="cursor">
                     <cursorShape>PointingHandCursor</cursorShape>
                    </property>
                    <property name="toolTip">
                     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;系统设置&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">
/*btnAddFriend*/
QPushButton#btnSetting
{
    background-image: url(:/style/blue/yunpan/header_icon_shezhi_n.png);
    background-position: center;
    background-repeat: no-repeat;
    border-radius: 3px;
}

QPushButton#btnSetting:hover
{
    background-image: url(:/style/blue/yunpan/header_icon_shezhi_h.png);
    background-position: center;
    background-repeat: no-repeat;
    border-radius: 3px;
}
</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="labelSplitter">
                    <property name="minimumSize">
                     <size>
                      <width>16</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btnMini">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>16</width>
                      <height>16</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16</width>
                      <height>16</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btnMax">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>16</width>
                      <height>16</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16</width>
                      <height>16</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btnClose">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>16</width>
                      <height>16</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16</width>
                      <height>16</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton
{ 
	background-image: url(:/style/blue/yunpan/yunpan_icon_close_n.png);
background-position:center;
background-repeat:no-repeat;
border-radius:3px;
}
QPushButton:hover
{
	background-image: url(:/style/blue/yunpan/header_icon_close_h.png);
background-position:center;
background-repeat:no-repeat;
border-radius:3px;
}</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="homeHLayout2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="NavWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>168</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>168</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget#NavWidget
{
	background-color: rgb(255, 255, 255);
}</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPushButton" name="homeButton">
             <property name="text">
              <string>home</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="webButton">
             <property name="text">
              <string>web</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <widget class="QWidget" name="page">
           <property name="styleSheet">
            <string notr="true">QWidget#page
{
	background-color: rgb(255, 255, 255);
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="home" native="true">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <layout class="QHBoxLayout" name="homeHLayout">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_2">
           <property name="styleSheet">
            <string notr="true">QWidget#page_2
{
	background-color: rgb(255, 255, 255);
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
